Доработать функциональность по ТЗ 6.

7Jenko opened the pull request:
1: Сделать историю посещений неограниченной по размеру.
2: Убрать повторные просмотры в истории.
3: Добавить дополнительные тесты менеджера задач.

7Jenko committed:
sprint_6-solution
1: Сделать историю посещений неограниченной по размеру.
2: Убрать повторные просмотры в истории.
3: Добавить дополнительные тесты менеджера задач.

7Jenko committed:
sprint_6-solution
Исправить HistoryManagerTest.java:21:5: 'METHOD_DEF' should be separated from previous line.

7Jenko committed:
sprint_6-solution
Исправить WhitespaceAround: '{': TaskManagerTest.java:74:43;
TaskManagerTest.java:85:42;
TaskManagerTest.java:97:47;
TaskManagerTest.java:108:53;
TaskManagerTest.java:119:48.
Исправить TaskManagerTest.java:96:5: 'METHOD_DEF' should be separated from previous line.

7Jenko committed:
sprint_6-solution
Исправить WhitespaceAround: '{': TaskManagerTest.java:74:43;
InMemoryTaskManager.java:22:29;
InMemoryTaskManager.java:60:32;
InMemoryTaskManager.java:71:32;
InMemoryTaskManager.java:88:35.

7Jenko committed:
sprint_6-solution
Исправить Main.java:63:5: 'METHOD_DEF' should be separated from previous line.

7Jenko committed:
sprint_6-solution
Исправить Subtask.java:21:27: WhitespaceAround: '{' is not preceded with whitespace.

7Jenko committed:
sprint_6-solution
Исправить find: ‘./test’: No such file or directory.

7Jenko committed:
sprint_6-solution
Исправить импорты тестов.

7Jenko committed:
sprint_6-solution
Исправить папка tests на test.

7Jenko committed:
sprint_6-solution
Переместить папка test.

7Jenko committed:
sprint_6-solution
Переместить папка test.

7Jenko committed:
sprint_6-solution
Переименовать test.

7Jenko committed:
sprint_6-solution
Переместить папку test.

7Jenko committed:
sprint_6-solution
Переместить папку test.

7Jenko committed:
sprint_6-solution
Переместить папку test.

7Jenko committed:
sprint_6-solution
Переместить папку test.

7Jenko committed:
sprint_6-solution
Исправить тесты на successful.

slavrikk commented on src/com/yandex/app/service/InMemoryHistoryManager.java:
```diff
@@ -1,26 +1,77 @@
     @Override
     public List<Task> getHistory() {
-        return List.copyOf(list);
+        return List.copyOf(getTasks());
```
`List.copyOf` излишне, так как метод `getTasks()` и так возвращает новую коллекцию

slavrikk commented on src/com/yandex/app/service/InMemoryHistoryManager.java:
```diff
@@ -1,26 +1,77 @@
+
+    @Override
+    public void remove(int id) {
+        removeNode(history.get(id));
```
Следует так же удалить ноду из мапы

7Jenko committed:
sprint_6-solution
Исправить метод getHistory в классе InMemoryHistoryManager.
Исправить метод remove в классе InMemoryHistoryManager.

7Jenko commented:
@slavrikk Привет, могу прожимать merge pull request?

slavrikk commented:
если апрувлю, то да)

7Jenko commented:
понял, спасибо)
первый раз все-таки, на всякий случай уточнил
